### PR TITLE
Same content style in each project tab

### PR DESCRIPTION
**Pull request by kilchenmann**
This PR adds the same toolbar to team viewer, to list viewer and to ontology viewer.

It includes already the ontology forms to create ontologies and to add resource classes, but they are disabled because of missing methods in @knora/api. The forms will be part in the next PR #229

### CODE REVIEW

**Comment by subotic** [comment]
@kilchenmann This is a huge and longrunning PR. Can you please try to keep PRs smaller and more focused in the future.

**Comment by kilchenmann** [comment]
@subic Yes I know it was growing to big. I will do smaller PRs in the future.

File: src/app/project/collaboration/add-user/add-user.component.html [hunk 1]
```diff
@@ -38,16 +40,32 @@ <h2 class="mat-title">{{"appLabels.form.user.title.add2project" | translate}}</h
 
                     </mat-form-field>
 
-                    <button mat-stroked-button [disabled]="!selectUserForm.valid" class="add-new" (click)="openDialog('createUser')">Or create new one</button>
-                    <button mat-button [disabled]="!selectUserForm.valid" class="add-new-mobile" (click)="openDialog('createUser')">
+                    <!-- <button mat-raised-button color="primary" [disabled]="!selectUserForm.valid" class="add-new"
+                        (click)="openDialog('createUser')">New user</button>
```
Comment (flavens) [inline src/app/project/collaboration/add-user/add-user.component.html]: @kilchenmann these commented lines can be deleted

File: src/app/project/list/list.component.html [hunk 2]
```diff
@@ -2,59 +2,113 @@
...
+
+    </div>
+
+
+    <!-- list of active lists -->
+    <!--
         <mat-accordion class="lists-list">
 
-            <mat-expansion-panel *ngFor="let list of projectLists; let i = index" class="list" (closed)="openPanel = -1" (afterExpand)="openPanel = i" [expanded]="projectLists.length === 1 || openPanel === i">
+            <mat-expansion-panel *ngFor="let list of lists; let i = index" class="list" (closed)="openPanel = -1"
```
Comment (flavens) [inline src/app/project/list/list.component.html]: @kilchenmann this code block can be deleted

Comment (kilchenmann) [reply src/app/project/list/list.component.html]: Done

**Pull request merged**